src/commands/System/Admin/update.ts

```diff
@@ -98,4 +109,9 @@ export default class extends SkyraCommand {
 		}
 	}
 
+	private cutText(input: string, ...extraInput: string[]) {
+		const cutLength = extraInput.length ? 1950 / (extraInput.length + 1) : 1950;
+		return cutText(input, cutLength);
+	}
```

Comment:
What I meant is that `extraInput.length` is always 1, because you call this method always with two arguments. Also, I'm not sure what this method does, some comments would help me understand it.